uqcsbot/scripts/repo.py

```diff
@@ -1,10 +1,58 @@
 from uqcsbot import bot, Command
 
+repos = {
+    "coc": "https://github.com/UQComputingSociety/code-of-conduct",
+    "constitution": "https://github.com/UQComputingSociety/constitution",
+    "cookbook": "https://github.com/UQComputingSociety/cookbook",
+    "desgin": "https://github.com/UQComputingSociety/design",
+    "events": "https://github.com/UQComputingSociety/events",
+    "inviter": "https://github.com/UQComputingSociety/slack-invite-automation",
+    "minutes": "https://github.com/UQComputingSociety/minutes",
+    "shirts": "https://github.com/UQComputingSociety/shirts",
+    "signup": "https://github.com/UQComputingSociety/signup",
+    "uqcsbot": "https://github.com/UQComputingSociety/uqcsbot",
+    "website": "https://github.com/UQComputingSociety/website"
+}
+
 
 @bot.on_command("repo")
 def handle_repo(command: Command):
     """
     `!repo` - Returns the url for the uqcsbot repo.
     """
-    bot.post_message(command.channel_id, "UQ Computing Society :github: Repository\n"
-                     "https://github.com/UQComputingSociety/uqcsbot")
+    # Setup for message passing
+    channel = bot.channels.get(command.channel_id)
+    # Read the commands provided
+    command_args = command.arg.split() if command.has_arg() else []
+
+    # Checks for the list command
+    is_list_output = False
+    if '--list' in command_args:
```

Comment:
Could this be simplified into a single if `'-l' in command_args or '--l' in command args` ?